src/pages/PreviousRuns.js

```diff
@@ -1,8 +1,244 @@
+        h(AutoSizer, [
+          ({ width, height }) => h(FlexTable, {
+            'aria-label': 'previous runs',
+            width, height, sort,
+            rowCount: paginatedPreviousRuns.length,
+            noContentMessage: 'Nothing here yet! Your previously run workflows will be displayed here.',
+            hoverHighlight: true,
+            columns: [
+              {
+                size: { basis: 500 },
+                field: 'workflowName',
+                headerRenderer: () => h(Sortable, { sort, field: 'workflowName', onSort: setSort }, ['Workflow Name']),
+                cellRenderer: ({ rowIndex }) => {
+                  return h(TextCell, [paginatedPreviousRuns[rowIndex].workflowName])
+                }
+              },
+              {
+                size: { basis: 150, grow: 0 },
+                field: 'workflowStatus',
+                headerRenderer: () => h(Sortable, { sort, field: 'workflowStatus', onSort: setSort }, ['Status']),
+                cellRenderer: ({ rowIndex }) => {
+                  return h(TextCell, [paginatedPreviousRuns[rowIndex].workflowStatus])
+                }
+              },
+              {
+                size: { basis: 80, grow: 0 },
+                field: 'workflowInputs',
+                headerRenderer: () => 'Inputs',
+                cellRenderer: ({ rowIndex }) => {
+                  return div({ style: { width: '100%', textAlign: 'center' } }, [
+                    h(Link, { onClick: () => setViewInputsId(rowIndex) }, ['View'])
+                  ])
+                }
+              },
+              {
+                size: { basis: 250, grow: 0 },
+                field: 'submissionTimestamp',
+                headerRenderer: () => h(Sortable, { sort, field: 'submissionTimestamp', onSort: setSort }, ['Submitted']),
+                cellRenderer: ({ rowIndex }) => {
+                  return h(TextCell, [Utils.makeCompleteDate(paginatedPreviousRuns[rowIndex].submissionTimestamp)])
+                }
+              }
+            ]
+          })
+        ])
+      ]),
+      !_.isEmpty(sortedPreviousRuns) && div({ style: { bottom: 0, position: 'absolute', marginBottom: '1.5rem', right: '4rem' } }, [
+        paginator({
+          filteredDataLength: sortedPreviousRuns.length,
+          unfilteredDataLength: sortedPreviousRuns.length,
+          pageNumber,
+          setPageNumber,
+          itemsPerPage,
+          setItemsPerPage: v => {
+            setPageNumber(1)
+            setItemsPerPage(v)
+          },
+          itemsPerPageOptions: [10, 25, 50, 100, 250, 500, 1000]
+        })
+      ])
+    ]),
+    (viewInputsId !== undefined) && h(Modal, {
+      title: 'Inputs JSON',
+      width: 600,
+      onDismiss: () => setViewInputsId(undefined),
+      showCancel: false,
+      okButton:
+        h(ButtonPrimary, {
+          disabled: false,
+          onClick: () => setViewInputsId(undefined)
+        }, ['OK'])
+    }, [
+      h(TextArea, {
```

Comment:
If "raw json" is likely to be the way this information is presented for at least a little while, I'd say let's try to do our best to make it as readable as possible.

There's a cool json formatter in Terra UI that might be worth a look (see an example usage in the [Workflow Dashboard](https://github.com/DataBiosphere/terra-ui/blob/dev/src/pages/workspaces/workspace/jobHistory/WorkflowDashboard.js#L197))

Reply:
That is a good point. I will try adding that 👍

Reply:
I have updated it to use `ReactJson`. Thank you for the suggestion!

![Screen Shot 2022-07-25 at 7 23 15 PM](https://user-images.githubusercontent.com/16748522/180891022-9df84259-833a-4491-b606-dfd19e073d82.png)